internal/metastore/kv/rootcoord/kv_catalog.go

```diff
@@ -523,6 +523,11 @@ func dropPartition(collMeta *pb.CollectionInfo, partitionID typeutil.UniqueID) {
 
 func (kc *Catalog) DropPartition(ctx context.Context, dbID int64, collectionID typeutil.UniqueID, partitionID typeutil.UniqueID, ts typeutil.Timestamp) error {
 	collMeta, err := kc.loadCollection(ctx, dbID, collectionID, ts)
+	if err != nil && errors.Is(err, merr.ErrCollectionNotFound) {
```

Comment:
`errors.Is(err, merr.ErrCollectionNotFound)` is enough.

Reply:
OK